docs-v2/pages/code/nodejs/index.mdx

```diff
@@ -352,6 +352,14 @@ In general, if you just need to make an HTTP request but don't care about the re
 
 You can return HTTP responses from [HTTP-triggered workflows](/workflows/steps/triggers/#http) using the [`$.respond()` function](/workflows/steps/triggers/#http-responses).
 
+## Invoke another workflow
+
+<Callout type="warning">
+This is an alpha feature and is subject to change without prior notice.
+</Callout>
+
+You can invoke another workflow in your workspace by running `await $.flow.trigger(workflowId, event)`, where `event` is an object containing the data to send to the workflow. Here's how you can [find your workflow's ID](/troubleshooting#where-do-i-find-my-workflows-id). This invocation remains within Pipedream — you don't need to configure a trigger, and the request does not leave the platform.
```

Comment:
I'd recommend making this a bit more structured and add a full code example like in other docs, e.g.

---

You can invoke another workflow in your workspace with `$.flow.trigger`:

```javascript
$.flow.trigger(
  workflowId, // Your Pipedream workflow ID, e.g. p_abc123
  payload 
)
```
[Find your workflow's ID here](/troubleshooting#where-do-i-find-my-workflows-id). 

This invokes the workflow directly — you don't need to configure a trigger, and the request does not leave the platform.